Tests/SentryTests/SentrySpanContextTests.swift

```diff
@@ -63,4 +63,27 @@ class SentrySpanContextTests: XCTestCase {
     func testSpanContextTraceTypeValue() {
         XCTAssertEqual(SpanContext.type, "trace")
     }
+    
+    func testSetTags() {
```

Comment:
`l`: As the tags are supposed to be thread-safe, can we please add tests for that. You can get inspired by https://github.com/getsentry/sentry-cocoa/blob/88e893eed4ba7e51e0d82834f18b7bec5b56905c/Tests/SentryTests/SentryScopeSwiftTests.swift#L224